content/docs/rfcs/62/README.md

```diff
@@ -0,0 +1,1077 @@
+
+### AcceptContactVerification
+
+A `AcceptContactVerification` signals that a verification request was accepted and includes a response to the challenge.
+
+
+```protobuf
+message AcceptContactVerification {
+  uint64 clock = 1;
+  string id = 2;
+  string response = 3;
+}
+```
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | clock | `uint64` | Clock value of the message | 
+| 2 | id | `string` | The verification request id |
+| 3 | response | `string` | The response for the challenge |
+
+### DeclineContactVerification
+
+A `DeclineContactVerification` declines a contact verification request.
+
+```protobuf
+message DeclineContactVerification {
+  uint64 clock = 1;
+  string id = 2;
+}
+```
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | clock | `uint64` | Clock value of the message | 
+| 2 | id | `string` | The verification request id |
+
+### CancelContactVerification
+
+A `CancelContactVerification` cancels a contact verification request.
+
+```protobuf
+message CancelContactVerification {
+  uint64 clock = 1;
+  string id = 2;
+}
+```
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | clock | `uint64` | Clock value of the message | 
+| 2 | id | `string` | The verification request id |
+
+
+### CommunityCancelRequestToJoin
+
+A `CommunityCancelRequestToJoin` cancels a pending request to join.
+
+```protobuf
+message CommunityCancelRequestToJoin {
+  uint64 clock = 1;
+  string ens_name = 2;
+  string chat_id = 3;
+  bytes community_id = 4;
+  string display_name = 5;
+}
+```
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | clock | `uint64` | Clock value of the message | 
+| 2 | ens_name | `string` | The ENS name of the account cancelling the request |
+| 3 | chat_id | `string` | The id of the chat |
+| 4 | community_id | `bytes` | The id of the community |
+| 5 | display_name | `string` | The display name of the account cancelling the request |
+
+### CommunityEditSharedAddresses
+
+A `CommunityEditSharedAddresses` message allows users to edit the shared accounts they've revealed when requesting to join a community.
+
+```protobuf
+message CommunityEditSharedAddresses {
+  uint64 clock = 1;
+  bytes community_id = 2;
+  repeated RevealedAccount revealed_accounts = 3;
+}
+```
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | clock | `uint64` | Clock value of the message | 
+| 2 | community_id | `bytes` | The id of the community |
+| 3 | revealed_accounts | `array<RevealedAccount>` | A list of revealed accounts |
+
+## Upgradability
+
+There are two ways to upgrade the protocol without breaking compatibility:
+
+- A node always supports accretion
+- A node does not support deletion of existing fields or messages, which might break compatibility
+
+## Security Considerations
+
+-
```

Comment:
this inherits security considerations of the key-exchange mechanism, waku-relay, no sender anonymity, etc